examples/index.html

```diff
@@ -81,9 +81,35 @@ <h1>Networked-Aframe Examples</h1>
     <p>View the above examples as AR scenes using a marker and AR.js</p> -->
 
     <br>
+    <br />
 
-    <p>Want to help make better examples? That would be awesome! <a href="https://github.com/networked-aframe/networked-aframe/blob/master/CONTRIBUTING.md#join-the-community-on-slack" target="_blank">Contact us on Slack</a>.</p>
+    <h2>Advanced Video Examples</h2>
+
+
+
+    <div>
+      <table>
+        <tr>
+          <td style="width:200px">
+          </td>
+          <td style="width:600px">
+                <a href="advanced-video-green-screen.html">Video with Green Screen</a>
+                <span>Extends Basic-Video by a shader that replaces green pixels with transparent ones</span>
+          </td>
+        </tr>
+      </table>
+    </div>
 
+
+
+
+
+
+    <br />
+    <br />
+    <h3>
+    <p>Want to help make better examples? That would be awesome! <a href="https://github.com/networked-aframe/networked-aframe/blob/master/CONTRIBUTING.md#join-the-community-on-slack" target="_blank">Contact us on Slack</a>.</p>
+    </h3>
```

Comment:
A p tag inside h3 is not valid. https://stackoverflow.com/questions/19779519/is-it-valid-to-have-paragraph-elements-inside-of-a-heading-tag-in-html5-p-insid

Reply:
I think we should add a representative screenshot for each example. It is easier thus to select. I will fix it. I am preparing an example.

Reply:
I didn't have any decent screenshot. I will make a better scene in the future